class MyException(Exception):
    pass

class MyIteratorNException(Exception):
    pass